internal/trace/logger.go

```diff
@@ -0,0 +1,27 @@
+package trace
+
+import "os"
+
+type DatadogLogger struct {
+	file *os.File
+}
+
+func NewDatadogLogger() (*DatadogLogger, error) {
+	file, err := os.Create("/tmp/upm.dd.log")
```

Comment:
Are we okay with only having the logs from the latest upm run. Probably fine, but we could parameterize by process id or something to make it more unique.

Reply:
I didn't want to potentially create a lot of files and make it harder to figure out which one is the latest. Any other down side to this?

Reply:
Nah. Fine to keep it like this.